### ISSUE TITLE
System Error when running Ganache 1.0.1 on win32 (eth_getBlockByNumber is crashing: cli.node.js:89944:5)

### ISSUE THREAD

**Issue by vsdigitall** [trufflesuite/ganache#165]
PLATFORM: win32
GANACHE VERSION: 1.0.1

EXCEPTION:
Error: Lost heartbeat from server process!
    at Timeout.setInterval [as _onTimeout] (/src/Services/Chain.js:86:30)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)

**Comment by vsdigitall** [comment]
Additional info:  this error caused by calling eth_getBlockByNumber, see log:

```
eth_getBlockByNumber
C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:74143
var Module;if(!Module)Module=(typeof Module!=="undefined"?Module:null)||{};var moduleOverrides={};for(var key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}var ENVIRONMENT_IS_WEB=false;var ENVIRONMENT_IS_WORKER=false;var ENVIRONMENT_IS_NODE=false;var ENVIRONMENT_IS_SHELL=false;if(Module["ENVIRONMENT"]){if(Module["ENVIRONMENT"]==="WEB"){ENVIRONMENT_IS_WEB=true}else if(Module["ENVIRONMENT"]==="WORKER"){ENVIRONMENT_IS_WORKER=true}else if(Module["ENVIRONMENT"]==="NODE"){ENVIRONMENT_IS_NODE=true}else if(Module["ENVIRONMENT"]==="SHELL"){ENVIRONMENT_IS_SHELL=true}else{throw new Error("The provided Module['ENVIRONMENT'] value is not valid. It must be one of: WEB|WORKER|NODE|SHELL.")}}else{ENVIRONMENT_IS_WEB=typeof window==="object";ENVIRONMENT_IS_WORKER=typeof importScripts==="function";ENVIRONMENT_IS_NODE=typeof process==="object"&&"function"==="function"&&!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_WORKER;ENVIRONMENT_IS_SHELL=!

TypeError: callback is not a function
    at C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:89944:5
    at C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:88596:7
    at C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:7542:16
    at iterate (C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:7750:24)
    at Object.async.forEachOfSeries.async.eachOfSeries (C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:7771:9)
    at Object.async.forEachSeries.async.eachSeries (C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:7704:22)
    at Object.decode (C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:88587:11)
    at C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:85340:21
    at C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:85778:12
    at C:\node\node_modules\ethereumjs-testrpc\build\cli.node.js:85543:19
```

**Comment by benjamincburns** [comment]
Thanks for reporting this. The lost heartbeat crash should be fixed as of our latest release, v1.0.2, which is out just now!

Unfortunately, you're also likely running afoul of trufflesuite/ganache-core#417. I'd suggest subscribing to that case for updates.
